sections/header.liquid

```diff
@@ -85,6 +85,9 @@
 <script src="{{ 'details-disclosure.js' | asset_url }}" defer="defer"></script>
 <script src="{{ 'details-modal.js' | asset_url }}" defer="defer"></script>
 <script src="{{ 'cart-notification.js' | asset_url }}" defer="defer"></script>
+{%- if settings.cart_type == "drawer" -%}
+  <script src="{{ 'cart-drawer.js' | asset_url }}" defer="defer"></script>
+{%- endif -%}
```

Comment:
So here what I'm seeing is that when I load the CSS in the header, I get a delay and the drawer first shows up as open. So I moved some of the styling to be in the `cart-drawer.liquid` section within a `<style>`. 
Let me know if you think I should go another way about it :)

Reply:
@ludoboludo Do we need to have all the styling in the `.liquid` file, in that case? Or can we move back non-critical properties to the `.css` file?

Would something like
```css
.drawer {
    visibility: hidden;
}
```
be enough to prevent this delay you noticed? Didn't test, just thinking out loud.

Reply:
Nah that's a good point. Probably works that way. Ill make an update locally and test.

Reply:
done, ready for another look 👍